tests/test_client.py

```diff
@@ -182,9 +195,22 @@ def test_list_accounts_failure(self, mock_get):
             client.list_accounts()
         except CoinbaseAdvancedTradeAPIError as page_error:
             self.assertDictEqual(page_error.error_dict, {
-                "error": "unknown",
-                "error_details": "some error details here",
-                "message": "some additional message here"
+                "success": False,
+                "failure_reason": "UNKNOWN_FAILURE_REASON",
+                "order_id": "",
+                "error_response": {
+                    "error": "INSUFFICIENT_FUND",
+                    "message": "Insufficient balance in source account",
+                    "error_details": "",
+                    "preview_failure_reason": "PREVIEW_INSUFFICIENT_FUND"
+                },
+                "order_configuration": {
+                    "limit_limit_gtc": {
+                        "base_size": "10000",
+                        "limit_price": ".19",
+                        "post_only": False
+                    }
+                }
```

Comment:
@gunjack-el not a release blocker at this time BUT will be great to create a set of unit tests for this client as we have for the REST API client.

Reply:
Just created an issue for this here: https://github.com/KmiQ/coinbase-advanced-python/issues/60